atomicfu-maven-plugin/build.gradle

```diff
@@ -23,6 +23,10 @@ def buildSnapshots = rootProject.properties['build_snapshot_train'] != null
 
 evaluationDependsOn(':atomicfu-transformer')
 
+def kotlinDevRepoUrl = (rootProject.properties['kotlin_repo_url'] ?: "" ) as String
```

Comment:
We should have a comment what this is for

Reply:
I've added a kdoc in KotlinAggregateBuild.kt